pulp_2to3_migration/app/pre_migration.py

```diff
@@ -185,8 +185,11 @@ def delete_removed_pulp2_content(content_model):
                 pulp2content_pb.total -= 1
                 pulp2detail_pb.total -= 1
                 continue
-
-        downloaded = record.downloaded if hasattr(record, 'downloaded') else False
+        is_downloaded = hasattr(record, 'downloaded') and getattr(record, 'downloaded')
+        if is_downloaded is not False:
+            downloaded = True
+        else:
+            downloaded = False
```

Comment:
I know the naming is hard here but it looks very confusing to me at least.
Do we need more variables? Is it worth just dealing with AND/OR? 
E.g.
`downloaded = hasattr(record, 'downloaded') and (record.downloaded is None or record.downloaded)`
Will that work?

If you prefer to keep if/else, let's brainstorm naming then, please.

Reply:
I don't mind switching to your suggestion, I find it hard sometimes to find proper variable name.